va/va.h

```diff
@@ -2019,16 +2031,18 @@ typedef struct _VAEncryptionSegmentInfo {
   uint32_t partial_aes_block_size;
   /** \brief  The length in bytes of the initial clear data */
   uint32_t init_byte_length;
-  /** \brief  This will be AES 128 counter for secure decode and secure
-   *  encode when numSegments equals 1 */
-  uint8_t aes_cbc_iv_or_ctr[16];
+  /** \brief  This will be AES counter for secure decode and secure encode
+   *  when numSegments equals 1 */
+  uint8_t aes_cbc_iv_or_ctr[64];
```

Comment:
so, not all aes_cbc_iv_or_ctr will be used, need to add some comments to describe it

Reply:
Updated